cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,63 @@
-describe('', () => {
-  before(() => {
-
+/// <reference types='cypress' />
+describe('Check article flow', () => {
+  let user;
+  let article;
+  beforeEach(() => {
+    cy.visit('/user/login');
+    cy.task('generateUser').then((userData) => {
+      user = userData;
+    });
+    cy.task('newArticle').then((newArticle) => {
+      article = newArticle;
+    });
   });
+  it('should create an article', () => {
+    cy.login(user.email, user.username, user.password);
+    cy.visit('');
+    cy.createArticle(article.title, article.description, article.body);
```

Comment:
```suggestion
    cy.createArticle(article.title, article.description, article.body);
```
why do you need to create an article before creating an article via UI?

Reply:
I`ve changed that